include/aws/common/error.h

```diff
@@ -19,11 +19,17 @@
 #include <aws/common/assert.h>
 #include <aws/common/exports.h>
 #include <aws/common/macros.h>
+#include <aws/common/package.h>
 #include <aws/common/stdint.h>
 
 #define AWS_OP_SUCCESS (0)
 #define AWS_OP_ERR (-1)
 
+#define AWS_ERROR_ENUM_STRIDE_BITS 10
```

Comment:
Maybe just a quick comment here so that the next reader doesn't have to math out what is happening here.

/* Create 1K ranges */ would even be enough